kie-server-parent/kie-server-services/kie-server-services-task-assigning/kie-server-services-task-assigning-planning/src/test/java/org/kie/server/services/taskassigning/planning/TaskAssigningPlanningKieServerExtensionTest.java

```diff
@@ -674,4 +683,46 @@ private void enableExtension() {
     private void disableExtension() {
         System.setProperty(KIE_TASK_ASSIGNING_PLANNING_EXT_DISABLED, "true");
     }
+
+    private String getCurrentKeyStore() {
+        String vendor = System.getProperty("java.vendor");
+        if (vendor.toUpperCase().contains("IBM")) {
+            return KEY_STORE_RESOURCE_IBM;
+        } else {
+            return KEY_STORE_RESOURCE;
+        }
+    }
+
+    /**
+     * Helper method for facilitating the generation of the KeyStore compatible with IBM jdk.
+     */
+    private void generateTaskAssigningIBMKeyStore() throws Exception {
```

Comment:
IDE suggests this is a dead code. As such, it may easily disappear in any refactoring to come. Do we need to keep the code, in other words, do we expect the keystore to change in the future? If so, how about moving the keystore creation into a separate class with a main method (still under the `test` directory structure)?

Reply:
I think it makes sense to keep it, it took time to investigate and do reverse engineering to understand how the original jBPM key store was created. If there are any "future" format issues we'll need this class for regenerate the keystore and test, etc.